### PR TITLE
Updates `Modernize existing .NET applications` eBook content to .net 5.0

### PR DESCRIPTION
**Pull request by sughosneo**
This PR updates the following pages :

- `Index`
- `Lift and shift existing .NET apps to Azure IaaS (Cloud Infrastructure-Ready)`
- `Migrate your relational databases to Azure`
- In the chapter : `Modernize existing .NET apps to Cloud-Optimized applications` :
	- `Index`
	- `Reasons to modernize existing .NET apps to Cloud-Optimized applications`
	- `What about Cloud-Native applications`
	- `Deploy existing .NET apps as Windows containers`
	- `When not to deploy to Windows Containers`
	- `When to deploy Windows Containers to Azure VMs (IaaS cloud)`
	- `When to deploy Windows Containers to Azure Container Instances (ACI)`
	- `Build resilient services ready for the cloud. Embrace transient failures in the cloud`
	- `Modernize your apps with monitoring and telemetry`
	- `Modernize your app's lifecycle with CI/CD pipelines and DevOps tools in the cloud`
	- `Migrate to hybrid cloud scenarios`
- `Walkthroughs and technical get started overview`

### CODE REVIEW

**Review by nishanil — COMMENTED** [review]
Looks good. Left a comment on Polly. 
I don't see an edition added to index. We should start numbering it from 5.0 and detail it out in the index page that the edition is in sync with the .NET 5.0 release cadence. Add changelogs to the wiki. We should remove the text "2nd edition" from cover and title.

File: docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/build-resilient-services-ready-for-the-cloud-embrace-transient-failures-in-the-cloud.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ You can use these techniques both in HTTP resources and in database resources. I
 
 When implementing resilient communications that access the database, depending on the version of .NET you are using, it can be straightforward (for example, [with Entity Framework 6 or later](/ef/ef6/fundamentals/connection-resiliency/retry-logic). It's just a matter of configuring the database connection). Or, you might need to use additional libraries like the [Transient Fault Handling Application Block](/previous-versions/msp-n-p/hh680934(v=pandp.50)) (for earlier versions of .NET), or even implement your own library.
 
-When implementing HTTP retries and circuit breakers, the recommendation for .NET is to use the [Polly](https://github.com/App-vNext/Polly) library, which targets .NET Framework 4.0, .NET Framework 4.5, and .NET Standard 1.1, which includes .NET Core support.
```
[the remark this answers is not in the document]
Comment (sughosneo) [reply docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/build-resilient-services-ready-for-the-cloud-embrace-transient-failures-in-the-cloud.md:32]: Yes, that's right. Included more specific information.

**Comment by sughosneo** [comment]
Hi @nishanil , thank you for your feedback. 
I have included all of your suggestions and it's ready now further review.

File: docs/architecture/modernize-with-azure-containers/index.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
 ---
 title: Modernize Existing .NET Applications With Azure Cloud and Windows Containers (2nd edition)
```
Comment (IEvangelist) [inline docs/architecture/modernize-with-azure-containers/index.md:2]:
```suggestion
title: Modernize Existing .NET Applications With Azure Cloud and Windows Containers (5th edition)
```

Per line number 10 below...

File: docs/architecture/modernize-with-azure-containers/index.md [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
 description: Learn to lift and shift and modernize existing applications to the Azure cloud and containers with this e-book.
-ms.date: 04/28/2018
+ms.date: 12/28/2020
 ---
 # Modernize existing .NET applications with Azure cloud and Windows Containers (2nd edition)
```
Comment (IEvangelist) [inline docs/architecture/modernize-with-azure-containers/index.md:6]:
```suggestion
# Modernize existing .NET applications with Azure cloud and Windows Containers (5th edition)
```

Per line number ten below...

File: docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/modernize-your-apps-with-monitoring-and-telemetry.md [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
+Application Insights is an extensible Application Performance Management (APM) service for web developers who work on multiple platforms. Use it to monitor your live web application. `Application Insights` automatically detects performance anomalies. It includes powerful analytics tools to help you diagnose issues, and to help you understand what users actually do with your app. Application Insights is designed to help you continuously improve performance and usability. It works for apps on a wide variety of platforms, including .NET, Node.js, and J2EE, whether hosted on-premises or in the cloud. Application Insights integrates with your DevOps processes, and has connection points to various development tools.
```
Comment (IEvangelist) [inline docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/modernize-your-apps-with-monitoring-and-telemetry.md:12]:
```suggestion
Application Insights is an extensible Application Performance Management (APM) service for web developers who work on multiple platforms. Use it to monitor your live web application. Application Insights automatically detects performance anomalies. It includes powerful analytics tools to help you diagnose issues, and to help you understand what users actually do with your app. Application Insights is designed to help you continuously improve performance and usability. It works for apps on a wide variety of platforms, including .NET, Node.js, and J2EE, whether hosted on-premises or in the cloud. Application Insights integrates with your DevOps processes, and has connection points to various development tools.
```

Application Insights is not code, therefore we should not mark it as such.

File: docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/modernize-your-apps-with-monitoring-and-telemetry.md [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
 
-Figure 4-10 shows an example of how Application Insights monitors your application and how it surfaces those insights to a dashboard.
+Figure 4-10 shows an example of how `Application Insights` monitors your application and how it surfaces those insights to a dashboard.
```
Comment (IEvangelist) [inline docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/modernize-your-apps-with-monitoring-and-telemetry.md:14]:
```suggestion
Figure 4-10 shows an example of how Application Insights monitors your application and how it surfaces those insights to a dashboard.
```

Same comment as above, if you're looking to emphasize this proper brand name you could either bold it, italicize it, or both - but do so sparingly.

File: docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/when-not-to-deploy-to-windows-containers.md [hunk 6]
```diff
@@ -27,7 +27,7 @@ Cases not supported in Windows Containers, as of May 2018:
 
 - Windows infrastructure roles (DNS, DHCP, DC, NTP, PRINT, File server, IAM etc.) are not supported scenarios.
 
-For additional not-supported scenarios and requests from the community, see the UserVoice forum for Windows Containers: <https://windowsserver.uservoice.com/forums/304624-containers>.
+For other not-supported scenarios and requests from the community, see the UserVoice forum for Windows Containers: <https://windowsserver.uservoice.com/forums/304624-containers>.
```
Comment (IEvangelist) [inline docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/when-not-to-deploy-to-windows-containers.md:30]:
```suggestion
For other nonsupported scenarios and requests from the community, see the UserVoice forum for Windows Containers: <https://windowsserver.uservoice.com/forums/304624-containers>.
```

File: docs/architecture/modernize-with-azure-containers/walkthroughs-technical-get-started-overview.md [hunk 7]
```diff
@@ -14,7 +14,7 @@ The following sections explain what each walkthrough is about, its objectives an
 
 The following get-started walkthroughs provide consistent and comprehensive technical guidance for sample apps that you can lift and shift by using containers, and then move by using multiple deployment choices in Azure.
 
-Each of the following walkthroughs uses the new sample eShopLegacy and eShopModernizing apps, which are available on GitHub at <https://github.com/dotnet-architecture/eShopModernizing>.
+Each of the following walkthroughs uses the new sample `eShopLegacy` and `eShopModernizing` apps, which are available on GitHub at <https://github.com/dotnet-architecture/eShopModernizing>.
```
Comment (IEvangelist) [inline docs/architecture/modernize-with-azure-containers/walkthroughs-technical-get-started-overview.md:17]:
```suggestion
Each of the following walkthroughs uses the new sample eShopLegacy and eShopModernizing apps, which are available on GitHub at <https://github.com/dotnet-architecture/eShopModernizing>.
```

The project names are used throughout the eBook without emphasis, why change that now? This is something that should be discussed as a group and agreed on before hand. However, this is not code... either italic, bold, or both... or not at all.

**Review by IEvangelist — CHANGES_REQUESTED** [review]
A few minor nits, ping me when this is updated and I'll approve it and get it merged. Thank you so much, this is AWESOME!

File: docs/architecture/modernize-with-azure-containers/walkthroughs-technical-get-started-overview.md (hunk 7 above)
Comment (sughosneo) [reply docs/architecture/modernize-with-azure-containers/walkthroughs-technical-get-started-overview.md:17]: I see. Makes sense

File: docs/architecture/modernize-with-azure-containers/index.md [hunk 8]
```diff
@@ -1,19 +1,23 @@
 ---
-title: Modernize Existing .NET Applications With Azure Cloud and Windows Containers (2nd edition)
+title: Modernize Existing .NET Applications With Azure Cloud and Windows Containers (5th edition)
 description: Learn to lift and shift and modernize existing applications to the Azure cloud and containers with this e-book.
-ms.date: 04/28/2018
+ms.date: 12/28/2020
 ---
-# Modernize existing .NET applications with Azure cloud and Windows Containers (2nd edition)
+# Modernize existing .NET applications with Azure cloud and Windows Containers (5th edition)
```
Comment (nishanil) [inline docs/architecture/modernize-with-azure-containers/index.md:6]: I suggest to drop the edition text from the title. We do have this mentioned below anyway.

Comment (sughosneo) [reply docs/architecture/modernize-with-azure-containers/index.md]: Yes, that makes sense as we are removing edition information from the cover page as well. Updated the title.

**Comment by sughosneo** [comment]
> A few minor nits, ping me when this is updated and I'll approve it and get it merged. Thank you so much, this is AWESOME!

Hi @IEvangelist, I have included all the suggestions. If you could do the final review and approve the changes. 

Thank you.

**Review by IEvangelist — APPROVED** [review]
Looks good to me, we'll :shipit: - thank you 🙏

**Pull request merged**